index.html

```diff
@@ -41,6 +43,191 @@ <h1 class="section-1-heading-mobile playfair-display-font font-color">Losalini R
+
+              <ul class="project-technologies font-white">
+                <li class="font-color mr-3">HTML/CSS</li>
+                <li class="font-color mr-3">Ruby on Rails</li>
+                <li class="font-color">JavaScript</li>
+              </ul>
+
+              <div class="see-this-project-btn-mobile">
+                <a href="#" class="poppins-font bg-color font-white">See this project <span class="material-icons ">arrow_forward</span></a>
+              </div>
+            </div>
+          </div>   
+          
+          <!--Project Card 5-->
+          <div class="card-5-mobile">
+            <img src="images/image7.jpg" alt="Hands on a macbook keyboard" class="card-5-img-mobile"/>
+
+            <!--Project description-->
+            <div class="project-description bg-color">
+              <h3 class="poppins-font font-white font-weight-lighter">Project name goes here</h3>
+
+              <ul class="project-technologies font-white">
+                <li class="font-color mr-3">HTML/CSS</li>
+                <li class="font-color mr-3">Ruby on Rails</li>
+                <li class="font-color">JavaScript</li>
+              </ul>
+
+              <div class="see-this-project-btn-mobile">
+                <a href="#" class="poppins-font bg-color font-white">See this project <span class="material-icons ">arrow_forward</span></a>
+              </div>
+            </div>
+          </div>
+
+          <!--Project Card 6-->
+          <div class="card-6-mobile">
+            <img src="images/image8.jpg" alt="Hands on a macbook keyboard" class="card-6-img-mobile"/>
+
+            <!--Project description-->
+            <div class="project-description bg-color">
+              <h3 class="poppins-font font-white font-weight-lighter">Project name goes here</h3>
+
+              <ul class="project-technologies font-white">
+                <li class="font-color mr-3">HTML/CSS</li>
+                <li class="font-color mr-3">Ruby on Rails</li>
+                <li class="font-color">JavaScript</li>
+              </ul>
+
+              <div class="see-this-project-btn-mobile">
+                <a href="#" class="poppins-font bg-color font-white">See this project <span class="material-icons ">arrow_forward</span></a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+     <!--Section 3 | About Section for mobile screens-->
+     <section class="section-3-mobile">
+      <h2 class="section-3-heading-mobile playfair-display-font font-color">About</h2>
+      <p class="section-3-paragraph-mobile poppins-font font-color">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+      
+      <div class="section-3-button-mobile">

```

Comment:
If you refer to Figma template 5, the about section has padding on the left and right sides, but I think you forgot to add it. I recommend adding some padding. Preferably follow the template provided. 😊